internal/gomod/dependencies.go

```diff
@@ -95,15 +105,59 @@ func parseGoListOutput(output, rootVersion string) (map[string]Module, error) {
 			module.Version = rootVersion
 		}
 
-		dependencies[importPath] = Module{
+		dependencies[importPath] = GoModule{
 			Name:    module.Name,
 			Version: cleanVersion(module.Version),
 		}
 	}
 
+	var thisModule jsonModule
+	if err := json.NewDecoder(strings.NewReader(modOutput)).Decode(&thisModule); err != nil {
+		return nil, err
+	}
+
+	if thisModule.GoVersion == "" {
+		return nil, errors.New("could not find GoVersion for current module")
+	}
+
+	setGolangDependency(dependencies, thisModule.GoVersion)
+
 	return dependencies, nil
 }
 
+// The repository to find the source code for golang.
+var golangRepository = "github.com/golang/go"
+
+func setGolangDependency(dependencies map[string]GoModule, goVersion string) {
+	dependencies[golangRepository] = GoModule{
+		Name: golangRepository,
+
+		// The reason we prefix version with "go" is because in golang/go, all the release
+		// tags are prefixed with "go". So turn "1.15" -> "go1.15"
+		Version: fmt.Sprintf("go%s", goVersion),
+	}
+}
+
+func GetGolangDependency(dependencies map[string]GoModule) GoModule {
+	return dependencies[golangRepository]
+}
+
+// NormalizeMonikerPackage returns a normalized path to ensure that all
+// standard library paths are handled the same. Primarily to make sure
+// that both the golangRepository and "std/" paths are normalized.
+func NormalizeMonikerPackage(path string) string {
```

Comment:
Can we get unit tests for this new function?

Reply:
yup! done